ui/__tests__/components/search/search.test.js

```diff
@@ -3,8 +3,33 @@ import React from 'react';
 
 import { Search } from '../../../components/search/search';
 
-
 describe('<Search />', () => {
+  describe('actionPath()', () => {
+    it('returns /requirements/ by default', () => {
+      const router = { pathname: '/no-such-path', query: {} };
+      const rendered = shallow(<Search router={router} />);
+      const actual = rendered.instance().actionPath();
+      expect(actual).toEqual('/requirements/');
+    });
+    it('returns the pathname if "policies" is in the pathname', () => {
+      const router = { pathname: '/policies', query: {} };
+      const rendered = shallow(<Search router={router} />);
+      const actual = rendered.instance().actionPath();
+      expect(actual).toEqual('/policies');
+    });
+  });
+
+  describe('handleChange()', () => {
+    it('sets the "term" value of state', () => {
+      const router = { pathname: '/policies', query: {} };
+      const rendered = shallow(<Search router={router} />);
+      const event = { target: { value: 'fake-value' } };
+      const instance = rendered.instance();
+      instance.handleChange(event);
```

Comment:
Just curious: why do we call the method directly here, but use `simulate` later?

Reply:
No, no real reason but now that you mention it, using `simulate` gives us an added benefit of ensuring that the event handlers are set up with the proper elements. I'll start using that pattern from here on out.